waspc/data/Generator/templates/sdk/wasp/client/operations/internal/index.ts

```diff
@@ -6,10 +6,10 @@ import {
  } from 'superjson'
 
 // PRIVATE API
-export type OperationRoute = { method: HttpMethod, path: string }
+export type OperationRoute = { method: HttpMethod.Post, path: string }
 
 // PRIVATE API
-export async function callOperation(operationRoute: OperationRoute & { method: HttpMethod.Post }, args: any) {
+export async function callOperation(operationRoute: OperationRoute, args: any) {
```

Comment:
They all used `HttpMethod.Post` and this was causing type errors in `core.ts`, so I've changed it.